pkgs/by-name/kb/kbd/package.nix

```diff
@@ -13,10 +13,18 @@
   bzip2,
   xz,
   zstd,
+  bash,
   gitUpdater,
+  # This might be worthwhile to disable if the store lives on a compressed filesystem.
+  withCompression ? true,
+  # Allow disabling the obsolete unicode_start/unicode_stop utilities,
+  # which unfortunately pull in bash into the closure.
+  withObsolete ? true,
```

Comment:
Maybe `withScripts` is a more neutral wording

Reply:
Yes, that's better. Fixed!